components/google_calendar/sources/upcoming-event-alert/upcoming-event-alert.mjs

```diff
@@ -0,0 +1,112 @@
+import taskScheduler from "../../../pipedream/sources/new-scheduled-tasks/new-scheduled-tasks.mjs";
+import googleCalendar from "../../google_calendar.app.mjs";
+import { axios } from "@pipedream/platform";
+
+const docLink = "https://pipedream.com/docs/examples/waiting-to-execute-next-step-of-workflow/#step-1-create-a-task-scheduler-event-source";
+
+export default {
+  key: "google_calendar-upcoming-event-alert",
+  name: "Upcoming Event Alert",
+  description: `Triggers based on a time interval before an upcoming event in the calendar.
+    This source uses Pipedream's Task Scheduler.
+    [See here](${docLink}) for more info and instructions to connect your Pipedream account.`,
```

Comment:
Looks like this is rendered on a new line in the builder -- can you collapse to 2 lines?
![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/20780753/201421213-e1cc874a-db2c-4800-953f-3c1799cd047d.png)